# Description: Don't show goa in Unity, we use uoa there
# Upstream: the change is ubuntu specific
#
Index: gnome-control-center-3.3.91/panels/online-accounts/gnome-online-accounts-panel.desktop.in.in
===================================================================
--- gnome-control-center-3.3.91.orig/panels/online-accounts/gnome-online-accounts-panel.desktop.in.in   2012-03-05 15:04:55.000000000 +0100
+++ gnome-control-center-3.3.91/panels/online-accounts/gnome-online-accounts-panel.desktop.in.in        2012-03-06 17:52:14.907083003 +0100
@@ -7,7 +7,7 @@
 Type=Application
 StartupNotify=true
 Categories=GNOME;GTK;Settings;DesktopSettings;X-GNOME-Settings-Panel;X-GNOME-PersonalSettings;
-OnlyShowIn=GNOME;Unity;
+OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-control-center
 X-GNOME-Bugzilla-Component=Online Accounts
